wallet-new/src/Cardano/Wallet/API.hs

```diff
@@ -49,7 +51,7 @@ type DevAPI = "api" :> "development" :> Dev.API
 devAPI :: Proxy DevAPI
 devAPI = Proxy
 
-type WalletAPI = V0Doc :<|> V1Doc :<|> V0API :<|> V1API
+type WalletAPI = V0Doc :<|> V1Doc :<|> LoggingApi WalletLoggingConfig (V0API :<|> V1API)
```

Comment:
At least that's not too invasive when it comes the time to hook it in 😀 

Probably not needed now, but in the future we might think about renaming this one to better reflect that its purpose is not just the one of logging, but logging information securely. For now let's not bother as there might be further changes we do to this part of the code.